integrationTests/vm/txsFee/multiShard/scCallWithValueTransfer_test.go

```diff
@@ -0,0 +1,67 @@
+//go:build !race
+// +build !race
+
+package multiShard
+
+import (
+	"encoding/hex"
+	"math/big"
+	"testing"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-go/config"
+	"github.com/multiversx/mx-chain-go/integrationTests/vm"
+	"github.com/multiversx/mx-chain-go/integrationTests/vm/txsFee/utils"
+	vmcommon "github.com/multiversx/mx-chain-vm-common-go"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDeployContractAndTransferValueSCProcessorV1(t *testing.T) {
+	testDeployContractAndTransferValue(t, false)
+}
+
+func TestDeployContractAndTransferValueSCProcessorV2(t *testing.T) {
+	testDeployContractAndTransferValue(t, true)
+}
+
+func testDeployContractAndTransferValue(t *testing.T, scProcessorV2Enabled bool) {
+	testContextSource, err := vm.CreatePreparedTxProcessorWithVMsMultiShard(0, config.EnableEpochs{})
+	require.Nil(t, err)
+	defer testContextSource.Close()
+
+	configEnabledEpochs := config.EnableEpochs{}
+	if !scProcessorV2Enabled {
+		configEnabledEpochs.SCDeployEnableEpoch = 1000
```

Comment:
is this ok? Should have been the other flag. Also, instead of providing the bool, you can provide the epoch directly. One test will be with epoch 0, the other, with a random value (1000 is good enough)
Reason: I've commented out the fix for the processor v1 and the tests still pass

Reply:
fixed